### DISCUSSION TITLE
not able to connect to cluster mode disabled cluster

### DISCUSSION THREAD

**Discussion by shinchan16** [redis/go-redis#2533] [Q&A]
Hi,
This is my golang code which i m using to connect to redis cache cluster where cluster mode is disabled.

```go
rdb := redis.NewClusterClient(&redis.ClusterOptions{
		NewClient: func(opt *redis.Options) *redis.Client {
			opt.Addr = "usw02-dev-timeseries-v11-migration-001.usw02-dev-timeseries-v11-migration.jlpe80.usw2.cache.amazonaws.com:6379"
			return redis.NewClient(opt)
		},
	})
	fmt.Println("rdb", rdb)
	rdb.Set("1", "2", 5*time.Minute)
	rdb.Get("1")
	result, err := rdb.Ping().Result()
	if err != nil {
		fmt.Println("err", err)
		return
	} else {
		fmt.Println("result", result)
	}
```

the redis cluster is present in AWS account..

Please ping me where is the issue.

**Comment by monkey92t** [comment]
Are you using Redis Cluster to achieve read-write separation?

**Reply by monkey92t** [reply]
I'm not sure what your goal is, but by doing this, you are connecting to the same aws-redis node every time, so it doesn't seem necessary to use cluster mode.
